android/src/main/java/ru/innim/flutter_login_facebook/MethodCallHandler.java

```diff
@@ -89,6 +94,23 @@ private void logIn(List<String> permissions, Result result) {
         LoginManager.getInstance().logIn(_activity, permissions);
     }
 
+    private void expressLogin(final Result result) {
+        LoginManager.getInstance().retrieveLoginStatus(_activity.getApplicationContext(), new LoginStatusCallback() {
+            @Override
+            public void onCompleted(AccessToken token) {
+                result.success(Results.loginSuccess(token));
+            }
+            @Override
+            public void onFailure() {
+                result.success(Results.loginCancel());
+            }
+            @Override
+            public void onError(Exception exception) {
+                result.success(Results.loginCancel());
```

Comment:
We should send error to the result, and leave for the Dart app part to handle it.